docs/organizations/configuring-default-git-provider-integration-settings.md

```diff
@@ -0,0 +1,13 @@
+# Configuring default Git provider integration settings
+
+You can configure the default settings that Codacy uses to integrate with your Git provider when you add a new repository to Codacy. This enables you to apply the same settings across your organization repositories.
+
+To configure these default settings, go to the Git provider integration page for your organization on the left navigation sidebar.
```

Comment:
Just a minor detail, but would "open" be a better word to use here? It's more consistent with what we're using on other pages, for example [Managing repositories](https://docs.codacy.com/organizations/managing-repositories/).

```suggestion
To configure these default settings, open the Git provider integration page for your organization on the left navigation sidebar.
```

Reply:
I guess I can... I was avoiding the "open" action because I'm not explaining in detail the procedure, including the step to toggle the options... 
So I was just describing where the user should go to configure (and not exactly the steps to configure). 🙄 

Do you think I should add the step to toggle the options? I assumed it was intuitive enough... 🤔

Reply:
I don't think it's necessary to explicitly instruct the users to change the toggles.

For me personally, it makes more sense to talk about opening Codacy UI pages (instead of "going to" the pages). But I was just checking the Microsoft Style Guide and it actually recommends using "go to" when instructing users to navigate to a certain pace on the UI. :+1:

Reply:
So I will leave it as is, unless you consider that we should use "open" for consistency with the remaining documentation pages.
If we want to follow the MS Style Guide, then we should later review these occurrences across documentation. 🤓